keycloak-provisioning/terraform.tf

```diff
@@ -87,6 +82,68 @@ resource "keycloak_realm" "realm" {
       password = var.realm_smtp_auth_password
     }
   }
+
+  # Realm Settings > User profile tab
+  attributes = {
+    userProfileEnabled = true
+  }
+}
+
+resource "keycloak_realm_user_profile" "userprofile" {
```

Comment:
I think you should get `username`, `firstName`,`lastName` and `email` by default so I don't think you have to define them here.  i.e. you should only need to define `organization` and `department` - but needs checked I could be wrong.

Reply:
all four need to be stated again if they are to be included in the user profile